testsuite/MDAnalysisTests/core/test_accumulate.py

```diff
@@ -149,3 +153,141 @@ def test_total_mass_duplicates(self, group):
         with pytest.warns(DuplicateWarning) as w:
             assert_almost_equal(group2.total_mass(), ref)
             assert len(w) == 1
+
+
+class TestMultipole(object):
+    """Tests the functionality of *Group.*_moment() like dipole_moment
+    and quadrupole_moment.
+    """
+
+    @pytest.fixture(scope='class')
+    def u(self):
+        u = mda.Universe(PDB_multipole)
+        u.add_TopologyAttr(
+            "charges",
+            [
+                -0.076,
+                0.019,
+                0.019,
+                0.019,
+                0.019,  # methane [:5] 
```

Comment:
```suggestion
                0.019,  # methane [:5]
```
One thing picked up by flake8